src/Router.php

```diff
@@ -123,23 +124,39 @@ private function add_route($path, $cb, $method = "GET")
     }
 
     # Create a GET route
-    public function get($path, $cb) {
+    public function get($path = null, $cb) {
```

Comment:
Having `null` as a route is not user-friendly or visually pleasing, perhaps you could change it to simply take empty strings?

Reply:
Yes . You are right. I notice that after creating the pull request.. sorry my bad